apps/studio/data/tables/tables-query.ts

```diff
@@ -26,23 +24,34 @@ export async function getTables(
   let headers = new Headers()
   if (connectionString) headers.set('x-connection-encrypted', connectionString)
 
-  let queryParams = new URLSearchParams()
-  if (schema) queryParams.set('included_schemas', schema)
-  const searchStr = queryParams.toString()
+  let queryParams = {}
+  if (schema) {
+    queryParams = { included_schemas: schema }
+  }
 
-  const response = (await get(
-    `${API_URL}/pg-meta/${projectRef}/tables${searchStr ? `?${searchStr}` : ''}`,
-    {
-      headers: Object.fromEntries(headers),
-      signal,
-    }
-  )) as TablesResponse
+  const { data, error } = await get('/platform/pg-meta/{ref}/tables', {
+    params: {
+      header: {
+        'x-connection-encrypted': connectionString!,
+      },
+      path: {
+        ref: projectRef,
+      },
+      query: queryParams as any,
+    },
+    headers: Object.fromEntries(headers),
```

Comment:
probably no longer need Object.fromEntries i think
can just be `headers: headers`, or shorthand just `headers`